test/Test/nlp.jl

```diff
@@ -16,4 +16,5 @@ const MOI = MathOptInterface
         )
     )
     MOI.Test.test_linear_mixed_complementarity(mock, config)
+    MOI.Test.test_qp_mixed_complementarity(mock, config)
```

Comment:
You probably need to add a call to `MOI.Utilities.set_mock_optimize!` as above to set the optimal solution.